### DISCUSSION TITLE
Shell listening ports

### DISCUSSION THREAD

**Discussion by walterzilla** [steveiliop56/getashell#80] [General]
Just wondering if there's a prefixed assignable port range, if not it's maybe feasible to implement it as feature? Thanks!

**Comment by steveiliop56** [comment]
Hello!

Currently there is no assignable port range but getashell scans the port to make sure no service is running there.

**Comment by walterzilla** [comment]
Sure, I was aware of that behavior👍 
Though let's say that one could use such a feature in order to configure firewall nat rule set in advance if needed...
